src/main/java/uk/gov/companieshouse/search/api/mapper/ApiToResponseMapper.java

```diff
@@ -18,6 +20,12 @@ public ResponseEntity map(ResponseObject responseObject) {
                 return ResponseEntity.status(FOUND).body(responseObject.getData());
             case SEARCH_NOT_FOUND:
                 return ResponseEntity.status(NOT_FOUND).build();
+            case DOCUMENT_UPSERTED:
+                return ResponseEntity.status(OK).build();
+            case UPDATE_REQUEST_ERROR:
+                return ResponseEntity.status(BAD_REQUEST).build();
+            case UPSERT_ERROR:
+                return ResponseEntity.status(BAD_REQUEST).build();
```

Comment:
this 
`case UPDATE_REQUEST_ERROR:
                return ResponseEntity.status(BAD_REQUEST).build();
            case UPSERT_ERROR:
                return ResponseEntity.status(BAD_REQUEST).build();`

Can be written as 
          `case UPDATE_REQUEST_ERROR:
            case UPSERT_ERROR:
                return ResponseEntity.status(BAD_REQUEST).build();`